JSDemos/Demos/Form/ColumnsAdaptability/description.md

```diff
@@ -1,9 +1,10 @@
-The Form component can use different layouts depending on the screen width. To enable this feature, configure the following properties:     
-- Assign *"auto"* to the [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount) property to make the number of columns adapt to any screen size.
-- Specify the minimum column width in the [minColWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#minColWidth) property.
+The Form component can use different layouts depending on the screen width. Click one of the buttons on the right from the demo title to switch between horizontal and vertical orientations. To enable this feature, configure the Form as follows:
 
-You can also specify a limited number of screen types and how many columns the Form should have on each screen. Configure the following properties for this purpose:    
-- Specify the available screen sizes in the [screenByWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#screenByWidth) function. In this demo, this function returns two values. `'sm'` stands for screens narrower than 720 pixels and `'md'` for others.
-- Specify the [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCountByScreen) property to define how many columns a Form has depending on the screen width. This property has a higher priority than the **colCount** and **minColWidth** properties. 
+**Automatically Adjust the Number of Columns**  
+1. Assign *"auto"* to the [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount) property to make the number of columns adapt to any screen size. 
+1. Use the [minColWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#minColWidth) property to specify the minimum column width.
 
-Click one of the buttons on the right from the demo title to switch between horizontal and vertical orientations. Toggle the [CheckBox](/Documentation/ApiReference/UI_Components/dxCheckBox/) at the bottom to modify the **colCountByScreen** property value to see how the Form adapts to different screens. In this demo, you can see the difference on the vertical screen.
\ No newline at end of file
+**Fix the Number of Columns for Predefined Screen Sizes**       
+1. Use the [screenByWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#screenByWidth) function to specify the available screen qualifications depending on the screen width. In this demo, this function returns two values. `'sm'` stands for screens narrower than 720 pixels and `'md'` for others.
+1. Specify the [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCountByScreen) property to define how many columns a Form has depending on the available screen qualifications. This property has a higher priority than the **colCount** and **minColWidth** properties.   
```

Comment:
We already made it clear that these behaviors are mutually exclusive, so the last sentence is unnecessary.

```suggestion
1. Specify the [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCountByScreen) property to define how many columns a Form should contain depending on the available screen qualifiers. 

```